### ISSUE TITLE
Failed `shape_check == shape_b`: Shape of beta(inputs[1]) does not match. beta: (1, 100, 1, 1) != expected: (1, 100).

### ISSUE THREAD

**Issue by yuto51942** [sony/nnabla#814]
Hello.

I'm trying to convert a PyTorch model to onnx and then to NNabla, but when I convert `nn.BatchNorm1d()`, I get the error titled `nnabla_cli draw_graph`.

pytorch model:

```py
import os

import torch
from torch import nn


def main():
  model = nn.BatchNorm1d(100)
  input = torch.randn(20, 100)

  file_name = str(model.__class__.__name__)

  torch.onnx.export(model, input, os.path.join('onnx_model', file_name + '.onnx'))

  print(f'saved: {file_name}.onnx')


if __name__ == "__main__":
  main()
```

and onnx to nnabla convert:

```sh
nnabla_cli convert onnx_model/BatchNorm1d.onnx BatchNorm1d.nnp
```

When I try to check inside the model, an error occurs.

```sh
nnannabla_cli draw_graph BatchNorm1d.nnp
```

```log
2021-03-01 18:43:26,396 [nnabla][INFO]: Drawing: torch-jit-export
Traceback (most recent call last):
  File "/user/python/.venv/lib/python3.8/site-packages/nnabla/utils/cli/cli.py", line 141, in cli_main
    return_value = args.func(args)
  File "/user/python/.venv/lib/python3.8/site-packages/nnabla/utils/cli/draw_graph.py", line 44, in draw_graph_command
    graph = nnp.get_network(n)
  File "/user/python/.venv/lib/python3.8/site-packages/nnabla/utils/nnp_graph.py", line 498, in get_network
    return NnpNetwork(network_proto, self._params, batch_size, callback=callback)
  File "/user/python/.venv/lib/python3.8/site-packages/nnabla/utils/nnp_graph.py", line 415, in __init__
    self._create_function(f, callback, current_scope)
  File "/user/python/.venv/lib/python3.8/site-packages/nnabla/utils/nnp_graph.py", line 340, in _create_function
    outputs = function_instance(
  File "function.pyx", line 302, in nnabla.function.Function.__call__
  File "function.pyx", line 280, in nnabla.function.Function._cg_call
RuntimeError: value error in setup_impl
/Users/gitlab-runner/builds/9703d983/1/nnabla/builders/all/nnabla/src/nbla/function/./generic/batch_normalization.cpp:58
Failed `shape_check == shape_b`: Shape of beta(inputs[1]) does not match. beta: (1, 100, 1, 1) != expected: (1, 100).
```

- versions
  - Python: 3.8.0
  - torch: 1.7.1
  - nnabla: 1.16.0
  - nnabla-converter: 1.16.0
  - graphviz(pip): 0.16
  - graphviz(from brew): 2.46.1
- environment
  - macOS Catalina
  - CPU: x86

**Comment by TomonobuTsujikawa** [comment]
Thank you for reporting issue.
We have merged a fix for this issue by https://github.com/sony/nnabla/pull/828.
This fix is included in nnabla v1.18.0.

**Comment by yuto51942** [comment]
@TomonobuTsujikawa Thank you very much!!
